python/py-s3transfer/Portfile

```diff
@@ -15,9 +15,8 @@ long_description    ${description}
 homepage            https://github.com/boto/${python.rootname}
 master_sites        pypi:s/${python.rootname}
 distname            ${python.rootname}-${version}
-checksums           md5     976734d3ec36b8fc2c1b505a907c3259 \
```

Comment:
Why did you remove the md5 checksum?

Reply:
Since all other ports I've worked with had `rmd160` and `sha256` I was under the impression that it was unnecessary to include it as well. Even the MacPorts documentation (section 4.2) use `rmd160` and `sha256` when exemplifying the construction of a new port.

Reply:
Yes, `md5` is considered obsolete. However, it can be included if upstream publishes it for verification.

Reply:
Thanks, ok, I won't drop them any longer.

Reply:
Please keep it here too.

Reply:
Restored.